pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -7442,12 +7941,23 @@ def solve_model(model, options=None):
             results = solve_discrete_water_quality(model, opt, scaled=False)
         elif model.config.water_quality is WaterQuality.post_process:
             # option 2.2:
-            results = opt.solve(model, tee=True)
+            if opt.options["solver"] == "CPLEX":
+                results = opt.solve(model, tee=True)
+            else:
+                results = opt.solve(model, tee=True)
```

Comment:
```suggestion
```
This `if` / `else` block appears unneeded - the same code is in both the `if` block and the `else` block.